bc/subscription/tasks.py

```diff
@@ -101,7 +102,9 @@ def process_fetch_webhook_event(fwe_pk: int):
     document = download_pdf_from_cl(cl_document["filepath_local"])
 
     sponsorship = get_active_sponsorship()
+    sponsor_message = None
```

Comment:
Something to think about, and maybe fix eventually: This function is getting pretty "thick." In theory, this function should be very "thin," leaving just about all logic and look-ups to the task, so it can respond to the webhook event as quickly and simply as possible.

I think it'd be pretty easy to move most of this logic into the task, and make the task's signature much smaller. It's usually code-smell when you have five required parameters to a function.

We can fix it later, but it's something I noticed.

Reply:
I agree with you about the number of parameters. I'm not sure if you created the comment in the right function because this one requires just one parameter, I think you're referring to the function called `make_post_for_webhook_event`.

I also noticed you mentioned the webhook event in your comment but none of the functions in this file handle the webhook directly, all of them are used as functions that We invoke asynchronously in a worker (background) process after We store the data of the event.  You can find the functions that receive the webhook events in the [api_views.py file](https://github.com/freelawproject/bigcases2/blob/main/bc/subscription/api_views.py) inside the [subscription folder](https://github.com/freelawproject/bigcases2/tree/main/bc/subscription)

Reply:
> I think you're referring to the function called make_post_for_webhook_event.

Yeah, sorry, it wouldn't let me highlight unedited lines. 

> I also noticed you mentioned the webhook event in your comment

Oh good point. I'll take another look next time we're in this area of the code.